Sources/LightweightCharts/LightweightChartsModels/Options/Basics/BaseValueType.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+/**
+ * Represents a type of a base value of baseline series type.
+ */
+public enum BaseValueType {
+    case baseValuePrice(valuePrice: BaseValuePrice)
```

Comment:
I guess `valuePrice` can be omitted because `BaseValuePrice` and `baseValuePrice` provide enough context